src/main/kotlin/com/faforever/userservice/controller/OAuthController.kt

```diff
@@ -114,4 +119,14 @@ class OAuthController(
         }.flatMap { redirectUrl ->
             redirect(response, redirectUrl)
         }
+
+    @PostMapping("revokeSession")
+    @PreAuthorize("hasRole('" + FafRole.ADMIN_ACCOUNT_BAN + "') + hasRole('" + OAuthScope.ADMINISTRATIVE_ACTION + "')")
```

Comment:
hasRole won't work on OAuth scopes.
Also this is Kotlin. If you make it const val string, you don't need to concat the string, just user `hasRole('${fafRole.ADMIN_ACCOUNT_BAN}')`

Reply:
more magic